package.json

```diff
@@ -62,6 +62,7 @@
   "dependencies": {
     "lodash.curry": "^4.1.1",
     "node-polyglot": "^2.2.2",
+    "prop-types": "^15.6.1",
```

Comment:
`prop-type` should be a peerDependency.
Anyway, this should be done in another PR, can you revert `package.json` and `package-lock.json` ?

Reply:
Could you elaborate why it should be a peer dependency?

If it's a peer dependency, then it should also be declared in the devDependencies so that tests pass.

Reply:
> Could you elaborate why it should be a peer dependency?

This should be a peer dependency because it intended to be used in the user project.
When I say `it`, I'm talking about `react`, `redux` and `react-redux`. 

When you use `redux-polyglot`, you have to install `react`, `redux` and `react-redux` yourself.

It's the same for `react-redux` (please see [react-redux peerDependencies](https://github.com/reactjs/react-redux/blob/master/package.json#L46)).

Refs : 
  - https://nodejs.org/en/blog/npm/peer-dependencies/
  - https://stackoverflow.com/questions/26737819/why-use-peer-dependencies-in-npm-for-plugins?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa


-----------------

> If it's a peer dependency, then it should also be declared in the devDependencies so that tests pass.

Yes, you right, so you can add `prop-types` in devdep and peerdep.

------------------

Anyway, Let's finish this PR and discuss about peer deps in #94.